functions/billing/index.js

```diff
@@ -57,9 +57,22 @@ exports.stopBilling = async (pubsubEvent, context) => {
   if (pubsubData.costAmount <= pubsubData.budgetAmount) {
     return `No action necessary. (Current cost: ${pubsubData.costAmount})`;
   }
+  
+  if (!PROJECT_ID) {
+    return `No project specified`;
+  }
 
   await _setAuthCredential();
-  if (await _isBillingEnabled(PROJECT_NAME)) {
+  let billingEnabled;
+  try {
+    billingEnabled = await _isBillingEnabled(PROJECT_NAME);
+  }
+  catch (e) {
```

Comment:
Usually the catch is on the same line in Node:

`} catch (err) {`

Reply:
Sounds good, will do!

Reply:
Still needs to be formatted?